.github/workflows/test.yml

```diff
@@ -12,7 +12,7 @@ jobs:
 
     strategy:
       matrix:
-        system: [ubuntu, macos]
+        system: [ubuntu]
```

Comment:
Can we change this to `ubuntu-latest` while we're at it?

Reply:
Isn't it already running on latest via `${{ matrix.system }}-latest` on the next line?

Reply:
You are correct, missed that :facepalm: